templates/frontEndLayout.html

```diff
@@ -289,16 +299,218 @@
             <div  style = "text-align: left;">Cabbage</div>
         </td>
         <td>
-            <div class = "border">$2</div>
+            <div class = "border" onclick = "sell('c')">$2</div>
         </td>
         <td>
-            <div>2</div>
+            <div id = "ownedc">0</div>
         </td>
         <td>
-            <div class = "border">$12</div>
+            <div class = "border" onclick = "buy('c')">$12</div>
         </td>
     </tr>
 </table>
+<div id = "cash">CASH: $10</div>
+
+<script>
+var GAME_CONFIG = {
+    startingCash: 10,
+    seeds: {
+        a: {
+            name: "Alfalfa Sprouts",
+            image: "static/alfalfa.jpg",
```

Comment:
I don't think you need a general `image` value here.  You should be choosing one of the 3 sizes every time so we can switch to scaled assets later